packages/browser/src/Session.ts

```diff
@@ -134,7 +134,7 @@ export class Session extends EventEmitter {
    * @param init Optional parameters customizing the request, by specifying an HTTP method, headers, a body, etc. Follows the [WHATWG Fetch Standard](https://fetch.spec.whatwg.org/).
    */
   fetch = async (url: RequestInfo, init?: RequestInit): Promise<Response> => {
-    return this.clientAuthentication.fetch(this.info.sessionId, url, init);
+    return this.clientAuthentication.fetch(url, init);
```

Comment:
Huh. How was this not a major bug already?

Reply:
The `clientAuthentication.fetch` used to take three parameters, and to use the sessionId to look up the access token, so it was not expected to match the standard `fetch` signature. Is that what's your raising here ?

Reply:
Ah no I misread, I thought you replaced `this.info.sessionId` with `url`, i.e. that it used to just send a fetch to the session ID. Never mind!